checks.d/snmp.py

```diff
@@ -386,6 +388,16 @@ def submit_metric(self, name, snmp_value, tags=[]):
 
         metric_name = self.normalize(name, prefix="snmp")
 
+        if forced_type:
+            if forced_type.lower() == "gauge":
+                value = int(snmp_value)
+                self.gauge(metric_name, value, tags)
+                return
```

Comment:
could you have an 
```python
if ...
elif ...
else
```
syntax instead ? You would then raise an exception in the else statement as the state wouldn't be properly set up.